Makefile

```diff
@@ -3,12 +3,20 @@ include include.mk
 jenkins: STACK_OPTS += --test --bench --coverage
 export STACK_OPTS
 
-.PHONY: all clean clean-submodules distclean docs haddock jenkins kore stylish \
+.PHONY: all clean docs haddock jenkins kore k-frontend stylish \
         test test-kore test-k
 
 kore:
 	stack build $(STACK_BUILD_OPTS)
 
+k-frontend:
+	mkdir -p $(BUILD_DIR)
+	rm -rf $(K_DIR) $(K_NIGHTLY)
+	wget --output-document $(K_NIGHTLY) \
+	    $$(curl 'https://api.github.com/repos/kframework/k/releases' | jq --raw-output '.[0].assets[0].browser_download_url')
+	mkdir --parents $(K_DIR)
+	tar --extract --verbose --file $(K_NIGHTLY) --strip-components 1 --directory $(K_DIR)
```

Comment:
I don't think `tar` needs to be `--verbose`.

Reply:
Removed.